geomesa-nifi-processors/geomesa-nifi-processors-accumulo/pom.xml

```diff
@@ -126,6 +130,38 @@
             <version>2.8.47</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.accumulo</groupId>
+            <artifactId>accumulo-test</artifactId>
+            <version>${accumulo.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <!-- odd issue resolving some apacheds dependencies -->
+                <exclusion>
+                    <groupId>org.apache.directory.server</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-accumulo-datastore_${scala.binary.version}</artifactId>
+            <version>${geomesa.version}</version>
+            <classifier>tests</classifier>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.accumulo</groupId>
+            <artifactId>accumulo-minicluster</artifactId>
+            <version>${accumulo.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>jline</groupId>
```

Comment:
Hmmm... you needed jline as a test dependency?  Ohhhkkk...

(Just to let you know that I still read PRs;).)

Reply:
minicluster won't start without it :(